### PR TITLE
[ex19] Allow for placing the particle gun from the host.

### PR DESCRIPTION
**Pull request by hageboeck**
For TestEm3, the gun needs to be placed at negative x. When "estEm3" is encountered in the gdml name, the gun will shift itself.

@bernhardmgruber 
```
Run time: 0.153657

Mean energy deposit          9.87835 GeV <--
Mean number of gamma         5199.97
```
:+1:

### CODE REVIEW

File: examples/Example19/main.cpp [hunk 1]
```diff
@@ -254,9 +254,18 @@ int main(int argc, char *argv[])
   scoringPerVolume.energyDeposit      = energyDeposit;
   GlobalScoring globalScoring;
 
+  const bool testEm3 = gdml_file.find("estEm3") != std::string::npos;
```
Comment (hahnjo) [inline examples/Example19/main.cpp:257]: Matching for a specific name is very, very ugly. Can we make this more general, please?

Comment (bernhardmgruber) [reply examples/Example19/main.cpp:257]: Like I said on mattermost, I would just pass the values via command line arguments, like: `example19 -gunpos -220,0,0 -gundir 1,0,0`. This way we don't need to hardcode anything.

**Review by bernhardmgruber — APPROVED** [review]

File: examples/Example19/main.cpp (hunk 1 above)
Comment (hageboeck) [reply examples/Example19/main.cpp]: Better to put it in the readme. The defaults are correct for CMS.

Comment (bernhardmgruber) [reply examples/Example19/main.cpp]: Added to readme.

**Comment by bernhardmgruber** [comment]
Hmm, it seems I messed up the squashing of the git commit message :S

**Comment by hahnjo** [comment]
It's a weird behavior to push commits to a PR and then accept and merge it yourself...

**Comment by bernhardmgruber** [comment]
> It's a weird behavior to push commits to a PR and then accept and merge it yourself...

I agree, that's why I had @hageboeck have a look as well!

**Pull request merged**